src/main/resources/db/postgres/defaultDS/V1.0_017__pofim_169_foresporsel_endringer.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE FORESPOERSEL
+    ADD FORESPOERSEL_TYPE VARCHAR(50)
+        NOT NULL DEFAULT 'BESTILT_AV_FAGSYSTEM';
+
+ALTER TABLE FORESPOERSEL ALTER COLUMN SKJAERINGSTIDSPUNKT DROP NOT NULL;
```

Comment:
Sikker på at dette er en god løsning. Kunne man tenkt seg en egen tabell for AGI-IM.
Noe annet er at man kan hente en sak i bakgrunnen - uten å vise til AG.
Evt la FPsak opprette forespørsler for slike når IM ankommer.

Reply:
Vi diskuterte litt frem og tilbake om vi skulle ha en egen datamodell for det eller ikke. Vi endte opp med å ikke ta det siden de to tabellene ville blitt ganske like, alle feltene som ville vært nødvendig i ForespørselAGI trengs også i vanlig Forespørsel. Men vi kan godt ta en ny runde på dette hvis du har noen tanker rundt hvordan det løses. 

Når det gjelder å hente sak i bakgrunnen så ønsker vi vel ikke at fpinntektsmelding skal be om data fra fpsak. 

Er litt usikker på å få fpsak til å opprette forespørsler i disse tilfellene, jeg tenker i alle fall at flyten da kan bli ganske forvirrende.

@AnjaAalerud / @johannbmHar dere noen tanker eller er dere er uenig i noe av det jeg skriver?

Reply:
Se https://jira.adeo.no/browse/POFIM-169 for analyse av løsningsvalg